### PR TITLE
Testing

### PR DESCRIPTION
**Pull request by ericgaspar**
- Application had no https: redirection #92
- User and group were missing #90

### CODE REVIEW

**Comment by ericgaspar** [comment]
!testme

**Comment by ericgaspar** [comment]
!testme

**Comment by ericgaspar** [comment]
!testme

**Comment by ericgaspar** [comment]
@Jules-Bertholet I am not able to `pip install somepackage` anymore on a fresh install with all settings to default: 
```
ERROR: Could not install packages due to an EnvironmentError: [Errno 13] Permission denied: '/opt/jupyterlab/.venv/lib/python3.7/site-packages/numpy.libs'
Consider using the `--user` option or check the permissions.
```

**Comment by Jules-Bertholet** [comment]
@ericgaspar What user are you running `pip` as? In what context?

**Comment by ericgaspar** [comment]
I was expecting the same behaviour as in the previous version (the one in master branch):
- login as admin
- be able, as admin, to `pip install somepackage`

How can we install packages with pip?

**Comment by Jules-Bertholet** [comment]
@ericgaspar Are you talking about running commands from the terminal in Jupyter hub? It's been a while since I worked on this, so I'll need detailed reproduction steps to investigate

**Comment by ericgaspar** [comment]
Sorry for not beeing super clear. Here is some steps to reproduce:
-  install JupyterLab with `Enable terminal in the lab?` and `Allow users to install extensions?` set to `true`.
- login to JupyterLab as admin
- open a new notebook
- typing `pip install matplotlib` get the following error

Capture d’écran

I realized that  installing python modules wasn't working anymore in the testing branch while answering this issue https://github.com/YunoHost-Apps/jupyterlab_ynh/issues/94

File: scripts/install [hunk 1]
```diff
@@ -88,12 +98,16 @@ ynh_script_progression --message="Setting up source files..." --weight=64
...
-pushd $final_path
-
-PIPENV_VENV_IN_PROJECT="enabled" PIPENV_SKIP_LOCK=true  ynh_exec_warn_less python3 -m pipenv install jupyterlab==$jupyterlab_version jupyterhub notebook jupyterhub-ldapauthenticator pyzmq --three
-
+# Set permissions to app files
+mkdir -p "$final_path/.venv"
+chown -R $app:$app "$final_path"
+chmod -R g=u,g-w,o-rwx "$final_path"
+setfacl -nR -m g:$app.main:rx -m d:g:$app.main:rx "$final_path/.venv"
+setfacl -n -m g:$app.main:x "$final_path"
```
Comment (alexAubin) [inline scripts/install:106]: Uh this `$app.main` is weird, is this really what you meant ? Isnt this related to the permission bug being discussed ?

Comment (alexAubin) [reply scripts/install:106]:
```suggestion
setfacl -nR -m g:$app:rx -m d:g:$app:rx "$final_path/.venv"
setfacl -n -m g:$app:x "$final_path"
```

Bumpitybump, naively I think $app should be enough, but I don't know what thoughts were being the original $app.main :s

**Comment by ericgaspar** [comment]
!testme

**Comment by ericgaspar** [comment]
Here is some logs when trying to install JupyterLab: https://paste.yunohost.org/raw/izebelewin

**Comment by alexAubin** [comment]
```
 sudo -u eric env PATH=/opt/node_n/n/versions/node/15/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin /opt/yunohost/jupyterlab/.venv/bin/jupyter lab build --app-dir=/home/eric/.local/share/jupyterlab/lab
[...]
PermissionError: [Errno 13] Permission denied: '/home/eric/.local/share/jupyterlab'
```

Uh that's puzzling, not sure to understand why it would fail with permission error, $admin should have permission to write in its own home ? x_X

**Comment by ericgaspar** [comment]
!tastemetitus

**Comment by ericgaspar** [comment]
!tastemetitus

**Pull request closed without merging**